src/backend/db/migrations/1_initial-migrations.sql

```diff
@@ -70,12 +71,44 @@ CREATE TABLE "goal" (
   goal_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(), -- TODO: add index to allow reordering
   iep_id UUID REFERENCES "iep" (iep_id),
   description TEXT NOT NULL,
-  created_at TIMESTAMPTZ NOT NULL DEFAULT NOW()
+  created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+  category TEXT NOT NULL CHECK (category IN ('writing', 'reading', 'math', 'other'))
 );
 
 CREATE TABLE "subgoal" (
   subgoal_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(), -- TODO: add index to allow reordering
   goal_id UUID REFERENCES "goal" (goal_id),
   description TEXT NOT NULL,
-  created_at TIMESTAMPTZ NOT NULL DEFAULT NOW()
+  created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+  instructions TEXT,
+  target_max_attempts INTEGER --How many "questions" to administer in a single sitting
+
+  -- Different collection types may be added later:
+  -- collection_type TEXT NOT NULL CHECK (collection_type IN ('attempt', 'behavioral')),
+);
+
+CREATE TABLE "task" (
+  task_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+  subgoal_id UUID REFERENCES "subgoal" (subgoal_id),
+  assignee_id UUID REFERENCES "user" (user_id),
+  due_date TIMESTAMPTZ NOT NULL
 );
+
+CREATE TABLE "trial_data" (
+  id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+  subgoal_id UUID REFERENCES "subgoal" (subgoal_id),
+  created_by_user_id UUID REFERENCES "user" (user_id),
+  -- TODO: Possibly add optional reference to "task"
+  created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+
+  success_with_prompt INTEGER NOT NULL,
+  success_without_prompt INTEGER NOT NULL,
+
+  -- Optional depending on type of task
+  notes TEXT,
+  image_list TEXT[]
```

Comment:
it doesn't look like this is being used yet so I might remove it
this should point to `file_id`s, it's usually a bit of an antipattern to store arrays in sql

Reply:
I agree. I was thinking of having files reference trials, but kind of left it here as a placeholder since it wasn't essential yet. I'll remove it for now.